ui/public/locales/en/translation.json

```diff
@@ -5,7 +5,10 @@
       "message": "Something went wrong here....",
       "goBack": "Go back",
       "reload": "Reload",
-      "goHome": "Go to homepage"
+      "goHome": "Go to homepage",
+      "errors": {
+        "notFound": "The resource you are trying to access does not exist. This might be due to a typo in the URL or the resource might have been deleted or renamed."
```

Comment:
Can we change the path to _pages.error.notFound_ ?
As long as _"errors"_ only includes one item, I don't think it is necessary to nest it.

Reply:
Very good catch. Totally agree, I fixed that key (430822e50fb622057c1f477b139e2b58e5d8a7ad) and also found another translation key where we used plural instead of singular (a358d1cd58f0304e3b2974269e94eb634e9bbcee)